waspc/examples/todoApp/migrations/20230328100125_add_email_verified/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - Added the required column `isEmailVerified` to the `User` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "User" ADD COLUMN     "isEmailVerified" BOOLEAN NOT NULL;
```

Comment:
Do we also want to track other things, like number of emails sent, or last email send time, etc. so we don't allow users to spam emails via our apps?

Reply:
Also, maybe we should put this on a different object? Unsure, but that would avoid users having to add lots of fields to user if we just use an FK.